source/Modules/ConfigMgrCBDsc.ResourceHelper/ConfigMgrCBDsc.ResourceHelper.psm1

```diff
@@ -548,6 +548,50 @@ function Convert-CidrToIP
     }
 }
 
+function ConvertTo-CimCMScheduleString
+{
+    [CmdletBinding()]
+    [OutputType([Microsoft.Management.Infrastructure.CimInstance])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [String]
+        $ScheduleString,
+
+        [Parameter(Mandatory = $true)]
+        [String]
+        $CimClassName
+    )
+
+    $schedule = Convert-CMSchedule -ScheduleString $ScheduleString
+
+    if (-not [string]::IsNullOrEmpty($schedule.DaySpan))
```

Comment:
Should this be something different then , $schedule.DaySpan? I know that it will be there if it is a valid string but it read that it is specific to days.

Reply:
This is just a check to validate that the schedule isn't set to None.  If it is set to none DaySpan, HoureSpan, and MinutesSpan doesn't exist in the schedule